app/workers/parse_raw_event_job.rb

```diff
@@ -28,16 +28,32 @@ def perform(raw_event_id)
         raw_event_id: re.id
       )
 
+      producer = Poseidon::Producer.new(
+        ["#{LensServer.config.kafka.host}:#{LensServer.config.kafka.port}"],
+        "lens_bg_producer"
+      )
+
+      base_hash = {
+        application: application.id,
+        scenario: scenario.event_hash,
+        conroller: meta[:controller],
+        action: meta[:action]
+      }
+
+      messages = []
       details.each_with_index do |row, index|
-        new_request.events.create(
+        hash = base_hash.merge({
+          timestamp: Time.at(row[:start]).to_s(:iso8601),
           event_type: row[:type],
           content: row[:content],
           duration: row[:duration],
           started_at: row[:start],
           finished_at: row[:finish],
           position: index
-        )
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.